_data/simple-icons.json

```
@@ -1291,6 +1291,12 @@
             "hex": "F9DC3E",
             "source": "https://github.com/babel/website/blob/93330158b6ecca1ab88d3be8dbf661f5c2da6c76/website/static/img/babel-black.svg"
         },
+        {
+            "title": "Backblaze",
+            "hex": "D30035",
+            "source": "https://www.backblaze.com/company/news.html",
+            "guidelines": "https://www.backblaze.com/press/Backblaze_Brand_Guidance.pdf"
```

```suggestion
            "guidelines": "https://www.backblaze.com/company/news.html"
```

Use the page link. The resource link might be changed. Use the page link is more reliable.